core/java/iesi-core/src/main/java/io/metadew/iesi/script/action/wfa/WfaExecuteQueryPing.java

```diff
@@ -116,11 +116,21 @@ protected boolean executeAction() throws InterruptedException {
         boolean setRuntimeVariables = converSetRuntimeVariable(getSetRuntimeVariables().getValue());
         int timeoutInterval = convertTimeoutInterval(getTimeoutInterval().getValue());
         int waitInterval = convertWaitInterval(getWaitInterval().getValue());
-        Connection connection = ConnectionConfiguration.getInstance()
-                .get(new ConnectionKey(connectionName, this.getExecutionControl().getEnvName()))
-                .get();
+
+        Connection connection;
+        try {
+            connection = ConnectionConfiguration.getInstance()
+                    .get(new ConnectionKey(connectionName, this.getExecutionControl().getEnvName()))
+                    .get();
+        } catch (Exception e) {
+            throw new RuntimeException("Unknown connection name");
+        }
+
```

Comment:
instead of a try-catch clause, replace .get() with orElseThrow(() -> new RuntimeException(...))

Reply:
Corrected in next commit